extension/chrome/utils/Ws.js

```diff
@@ -105,7 +105,10 @@ export default class Ws {
 
   _executeCbsWithError () {
     logger.log('[WS] executing callbacks with error: ', this._callbacks);
-    this._callbacks.forEach(cb => cb('[WS] disconnected, cb cannot be called'));
+    for (const msgId in this._callbacks) {
```

Comment:
for of doesn't work on objects
Object.keys would give the keys, and the value is what's needed here

we could do
`Object.keys(this._callbacks).forEach(msgId => this._callbacks[msgId]('cb msg here ...')`

do u have a scenario in mind where the current code wouldn't work?